PyISY/Nodes/__init__.py

```diff
@@ -185,11 +206,18 @@ def parse(self, xml):
                             # Not all devices have properties
                             if len(nprop) > 0:
                                 nval = nprop[0].attributes['value'].value
-                                dimmable = '%' in \
-                                    nprop[0].attributes['uom'].value
+                                uom = nprop[0].attributes['uom'].value
+                                if uom in UNIT_OF_MEASURE:
+                                    units = UNIT_OF_MEASURE[unit]
+                                else:
+                                    units = uom.split('/')
+                                dimmable = '%' in units
+                                if 'prec' in nprop[0].attributes:
+                                    prec = nprop[0].attributes['prec'].value
                                 nval = int(nval.replace(' ', '0'))
                             self.insert(nid, nname, nparent,
-                                        Node(self, nid, nval, nname, dimmable),
+                                        Node(self, nid, nval, nname, dimmable,
+                                             uom=units, prec=prec),
```

Comment:
Also, units won't have a value if the ISY doesn't specify properties for the device. Both of these should have default values set around line 205.